### DISCUSSION TITLE
fd will not show pacnew files unless --no-ignore-vcs is used

### DISCUSSION THREAD

**Discussion by rieje** [sharkdp/fd#1325] [Q&A] [answered]
fd will not show pacnew files unless --no-ignore-vcs is used but I have no explicitly ignored them and have gone through what I think are all the .gitignore, .git/info/exclude, and ~/.config/git/ignore (doesn't exist for me) files. I run `fd pacnew /etc` which produces no results, while `fd pacnew /etc --no-ignore-vcs` does (I'm not in a git repo when I run this).

Any ideas? I must have missed something.

**Comment by tmccombs** [comment, marked as answer]
Is your /etc directory part of a git repo (for example are you using etckeeper)

**Reply by rieje** [reply]
That was it, totally forgot about it. Thank you.
